programs/programs/ks/pe/member.py

```
@@ -112,3 +112,64 @@ def member_value(self, member: HouseholdMember):
+        # msp_asset_eligible
+        dependency.spm.CashAssetsDependency,
+        # state
+        dependency.household.KsStateCodeDependency,
+        # Sends 40 quarters → is_premium_free_part_a=True → base_part_a_premium=$0
+        # QMB benefit value = Part B premium only (~99% of beneficiaries have free Part A)
+        dependency.member.MedicareQuartersOfCoverageDependency,
+        # is_medicaid_eligible (for QI exclusion) - override when user reports Medicaid
+        dependency.member.IsMedicaidEligibleDependency,
+        # Medicaid dependencies (for PolicyEngine's is_medicaid_eligible calculation)
+        *Medicaid.pe_inputs,
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)